src/main/java/com/github/maven_nar/cpptasks/compiler/CommandLineCompiler.java

```diff
@@ -134,17 +140,24 @@ protected void buildDefineArguments(CompilerDef[] defs, Vector<String> args) {
     }
     /**
      * Compiles a source file.
+     * @param commandPrefixes 
      * 
      */
     public void compile(CCTask task, File outputDir, String[] sourceFiles,
             String[] args, String[] endArgs, boolean relentless,
-            CommandLineCompilerConfiguration config, ProgressMonitor monitor)
+            CommandLineCompilerConfiguration config, ProgressMonitor monitor, boolean ccache)
```

Comment:
Both this and the `getCompiler()` rename are unnecessarily backwards-incompatible, and would therefore require a major version increase. The ccache only merits a minor version increase, though. Better leave redirects (for `getCompiler()` deprecated.